scripts/core_stats.sh

```diff
@@ -0,0 +1,39 @@
+zgrep 'Advancing LCL: \[seq=[0-9]*' $1 | \
```

Comment:
what is this script for? can you add some comments?

Reply:
It's part of log collection.  There are several script fragments of this nature.